tests/integration/domains/test_domain_records.py

```diff
@@ -112,6 +118,8 @@ def test_create_a_domain():
         "the list is not updated with new domain..",
     )
 
+    delete_target_id(target="domains", id=domain_id)
```

Comment:
A concern I have is that for some test cases like this one, if the previous assertion failed before reaching the cleanup part of the code, the created resources may remain on the cloud and required a manual cleanup... I think an ideal way is to put creation command invocation in a fixture, similar to this example below. Or can you think about another way for this kind of situation?
https://github.com/linode/linode-cli/blob/0e5208c8b3ce89eb3da020a320124bc4df709b72/tests/integration/test_obj_plugin.py#L90

Reply:
Thanks for this catch. You are right teardowns/cleanups should not happen after assertion